CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 Changes that are currently in development and have not been released yet.
 
+## [0.13.9](https://github.com/cossacklabs/themis/releases/tag/0.13.8), May 11th 2021
+
+**Hotfix for Apple platforms:**
+
+- `themis` for Carthage switched to using xcframeworks
+- Updated OpenSSL to the latest 1.1.1k for Carthage
+
+_Code:_
+
+- **Objective-C / Swift**
+
+  - `themis` for Carthage now loads OpenSSL as xcframework, and Carthege builds `themis` as scframework as well.
+  - Updated OpenSSL to the latest 1.1.1k for Carthage
+  - `create_xcframeworks.sh` script (used for SPM) no longer requires `Themis.xcodeproj` modifications because OpenSSL is now added as xcframework.
```

Comment:
`create_xcframeworks.sh` was this script changed? I don't see it in PR

Reply:
Themis.xcodeproj contained OpenSSL linked with Carthage. It was an older version of OpenSSL. SMP supported the newer OpenSSL version, so you had to go to the Themis.xcodeproj and add it before building the XCF. Now we can forget about all these manipulations because Carthage is adding the same XCF as SPM.